db/migrations/00016_add_receipts_fk_to_logs.sql

```diff
@@ -8,8 +8,13 @@ ALTER TABLE full_sync_logs
             REFERENCES full_sync_receipts (id)
             ON DELETE CASCADE;
 
+CREATE INDEX full_sync_logs_receipt
```

Comment:
Have we switched to an append-only migration workflow in vDB yet? Migrations like this one make me think we have, but migrations have also been changed as recently as a week ago.

Reply:
We have not, but migrations like this are a product of when we had made an attempt to do so earlier